### PR TITLE
Add update example for javascript client

### PR DESCRIPTION
**Pull request by nuhmanpk**
### Description
Added Update example for `client.update` method for javascript client

### Issues Resolved
None

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by Naarcha-AWS** [comment]
@nuhmanpk: Is this related to any specific version of OpenSearch?

**Comment by kolchfa-aws** [comment]
@Naarcha-AWS This PR is related to the Javascript client so I'm reassigning to myself

File: _clients/javascript/index.md [hunk 1]
```diff
@@ -327,6 +327,27 @@ var response = await client.search({
 ```
 {% include copy.html %}
 
+## Updating a document
+
+You can delete a document using the client's `update` method:
```
Comment (kolchfa-aws) [inline _clients/javascript/index.md:332]:
```suggestion
You can update a document using the client's `update` method:
```

**Review by kolchfa-aws — APPROVED** [review]
Thank you, @nuhmanpk!

**Pull request merged**